app/Http/Controllers/DashboardController.php

```diff
@@ -0,0 +1,135 @@
+        $messageSuccess = MessageHistory::dataChart($statisticParams, MessageHistoryStatus::SUCCESS);
+
+        $payloadFaildedQuantity = $this->getDataChart($payloadFailded, $period);
+        $payloadSuccessQuantity = $this->getDataChart($payloadSuccess, $period);
+        $messageFaildedQuantity = $this->getDataChart($messageFailed, $period);
+        $messageSuccessQuantity = $this->getDataChart($messageSuccess, $period);
+
+        $payloadFailedChart =  $this->convertArrayChart($payloadFaildedQuantity, $period);
+        $payloadSuccessChart = $this->convertArrayChart($payloadSuccessQuantity, $period);
+        $messageFailedChart = $this->convertArrayChart($messageFaildedQuantity, $period);
+        $messageSuccessChart = $this->convertArrayChart($messageSuccessQuantity, $period);
+        $dateChart = $this->convertArrayChart($period);
+
+        $payloadChartData = [
+            'failedCases' => array_sum($payloadFaildedQuantity),
+            'successCases' => array_sum($payloadSuccessQuantity),
+            'dateChart' => $dateChart,
+            'payloadFailedChart' => $payloadFailedChart,
+            'payloadSuccessChart' => $payloadSuccessChart
+        ];
+        $messageChartData = [
+            'failedCases' => array_sum($messageFaildedQuantity),
+            'successCases' => array_sum($messageSuccessQuantity),
+            'dateChart' => $dateChart,
+            'messageFailedChart' => $messageFailedChart,
+            'messageSuccessChart' => $messageSuccessChart
+        ];
+
+        return view('dashboard', ['countData' => $countData, 'payloadHistory' => $payloadChartData, 'messageHistory' => $messageChartData]);
+    }
+
+    function handleStatisticParam($statisticParams) {
+        if ($statisticParams == null) {
+            $statisticParams = ['fromDate' => date("01-m-Y"), 'toDate' => date("d-m-Y")];
+        }
+
+        if ($statisticParams['fromDate'] == null) {
+            $statisticParams['fromDate'] = date("01-m-Y");
+        }
+
+        if ($statisticParams['toDate'] == null) {
+            $statisticParams['toDate'] = date("d-m-Y");
+        }
+
+        return $statisticParams;
+    }
+
+    function getDateFromRange($start, $end, $format = 'd-m-Y') {
+        if ($start == null) {
+            $start = date("01-m-Y");
+        }
+
+        if ($end == null) {
+            $end = date("d-m-Y");
+        }
+        // Declare an empty array
+        $array = array();
+
+        // Variable that store the date interval
+        // of period 1 day
+        $interval = new DateInterval('P1D');
+
+        $realEnd = DateTime::createFromFormat('d-m-Y', $end);
+        $realEnd->add($interval);
+
+        $period = new DatePeriod(DateTime::createFromFormat('d-m-Y', $start), $interval, $realEnd);
+
+        // Use loop to store date into array
+        foreach($period as $date) {
+            $array[] = $date->format($format);
+        }
+
+        // Return the array elements
+        return $array;
+    }
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Squiz.Functions.MultiLineFunctionDeclaration.BraceOnSameLine</br>**Severity**: error</br>**File**: app/Http/Controllers/DashboardController.php</br>**Line**: 112</br>Opening brace should be on a new line